update-rules.json

```diff
@@ -930,5 +930,30 @@
       }
     },
 	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+     "handbrake": {
+     "url": "https://handbrake.fr/downloads.php",
+     "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='i686-Win_GUI-1.exe']"
```

Comment:
It looks like  the naming convention changes from version to version. The "-1" is not always included in the file name, so a more generic selector will be needed.